src/ragger/conftest/configuration.py

```diff
@@ -23,6 +25,16 @@ class OptionalOptions:
     # example: "./app/"
     APP_DIR=".",
 
+    # Use this parameter to point to the repository holding the build output of the app for each device
+    # This parameter defaults to the C-SDK build output ("build/")
+    # For RUST-SDK, should be set to "target"
+    BUILD_DIR="build",
+
+    # Use this parameter to point to the app binary
+    # This parameter defaults to the C-SDK binary name ("bin/app.elf")
+    # For RUST-SDK binary, should be for instance set to "release/app-name"
+    BIN_NAME="bin/app.elf",
+
```

Comment:
This is what I have initially thought about but it will imply that all applications using already `Ragger` will have to be modified (at least their `ledger_app.toml`), no ?

Reply:
:thinking: I don't see any issue yet. The applications will need to have a `ledger_app.toml` file, but except that it should be ok, manifest for Rust and C apps are the same. Did you encountered specific issues?

Reply:
The missing bit will be, if I understand correctly, the `elf` name when using the Rust SDK.
Is it the only binary present in the `release` directory? Is there a way to discriminate it from other files that could be present?

Reply:
For now I guess parsing the `Cargo.toml` and fetching the `toml["package"]["name"]` seems like a viable workaround, what do you think?

Reply:
Draft in #172

Reply:
Ok so if I understand correctly, your proposal is to add the path to binary artifacts produces when building an app (from C SDK or Rust SDK) in the manifest, or am I missing anything ? 
For Rust SDK, `cargo ledger build` produces several binaries (`.hex`, `.apdu` and `elf`) into `target/{device}/release/` where device is either `nanos` or `nanosplus` or `nanox` (soon `stax` and `europa`).

Regarding binaries name, it is retrieved from package name (set in `Cargo.toml`) and for instance, with app-boilerplate-rust, we will have:
- `elf` => `app-boilerplate-rust` (without extension),
- `.hex` => `app.hex` (should be changed into `app-boilerplate-rust.hex` for consistency),
- `.apdu` => `app-boilerplate-rust.apdu`

So initially my PR aims at enabling to use `Ragger` tests (speculos backend) with binaries produces either from C-SDK or Rust-SDK, without having to copy manually binaries from a Rust build tree into a C-SDK compliant build tree (this is the WoW nowadays).

Reply:
Nothing is to be added in the manifest, all information seems already accessible with users needing to add anything in conf file.
#172 highlights this. I've not altered any manifest, just used `ledgered` to infer the binary location. It seems to work on boilerplate C **and** Rust

Reply:
Yes it seems to be ok, thanks !